googleapiclient/errors.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, reason, resp=None, content=None):
     self.reason = reason
 
   def __repr__(self):
+    if self.resp is not None and hasattr(self.resp, 'status'):
```

Comment:
You should be aware that there is a movement gaining traction saying that [`hasattr` should never be used](http://doc.bazaar.canonical.com/developers/code-style.html#hasattr-and-getattr).